pyproject.toml

```diff
@@ -47,6 +49,16 @@ warn_return_any = true
 warn_redundant_casts = true
 namespace_packages = true
 
+[tool.pyright]
+include = ["crescent", "examples"]
+exclude = ["tests"]
+ignore = ["examples/basic/basic.py", "examples/basic/autocomplete.py"]
```

Comment:
dont ignore these

Reply:
They're ignored because:
1. Pyright is set to 3.8, and there is an example to import Annotated from 3.9
2. dot_env has type issues

Do you still want to remove it? In fact I think mypy has issues too, it's just not checked in CI.

Reply:
I agree with circuit on this

Reply:
Is the comment saying to import from typing 3.9+ fine?

Reply:
yes, only in examples